### ISSUE TITLE
No file, still returns Valid

### ISSUE THREAD

**Issue by Asbjoedt** [opf-labs/odf-validator#13]
Hi
I try to input a folder and it gives error, that it is not an existing file.
```
APP-1: [INFO] Validating C:\DB\_Testdata\Regneark\ODFTestData\ODFTestData\Spreadsheets\.fods. 
SYS-1: [FATAL] Path C:\DB\_Testdata\Regneark\ODFTestData\ODFTestData\Spreadsheets\.fods IS NOT an existing file.
No messages.
VALID, no errors and no warnings found.
```

This is good behaviour, but it also writes:
```
No messages.
VALID, no errors and no warnings found.
```

I don't think it should write any of this this, especially not "VALID". What do you think?

I also tried running through a .fods file. Now it gives many good messages and some are even ERRORS, but it still writes at the end:

```
No messages.
VALID, no errors and no warnings found.
```

This is the full output:
```
APP-1: [INFO] Validating C:\DB\_Testdata\Regneark\ODFTestData\ODFTestData\Spreadsheets\.fods\FODS_Embedded fonts.fods. 
ZIP-1: [INFO] Path C:\DB\_Testdata\Regneark\ODFTestData\ODFTestData\Spreadsheets\.fods\FODS_Embedded fonts.fods is not a ZIP file. 
PKG-3: [INFO] Open Document Format version 1.3 detected.
PKG-4: [INFO] Document MIMETYPE declared: application/vnd.oasis.opendocument.spreadsheet
XML-1: [ERROR] SAX Parser Exception: attribute "loext:font-style" not allowed here; expected attribute "xlink:actuate", "xlink:href" or "xlink:type".
XML-1: [ERROR] SAX Parser Exception: attribute "loext:font-weight" not allowed here; expected attribute "xlink:actuate", "xlink:href" or "xlink:type".
XML-1: [ERROR] SAX Parser Exception: element "svg:font-face-uri" missing required attributes "xlink:href" and "xlink:type".
XML-1: [ERROR] SAX Parser Exception: element "office:binary-data" not allowed here; expected the element end-tag or element "svg:font-face-format".
XML-1: [ERROR] SAX Parser Exception: attribute "loext:font-style" not allowed here; expected attribute "xlink:actuate", "xlink:href" or "xlink:type".
XML-1: [ERROR] SAX Parser Exception: attribute "loext:font-weight" not allowed here; expected attribute "xlink:actuate", "xlink:href" or "xlink:type".
XML-1: [ERROR] SAX Parser Exception: element "svg:font-face-uri" missing required attributes "xlink:href" and "xlink:type".
XML-1: [ERROR] SAX Parser Exception: element "office:binary-data" not allowed here; expected the element end-tag or element "svg:font-face-format".
XML-1: [ERROR] SAX Parser Exception: attribute "loext:font-style" not allowed here; expected attribute "xlink:actuate", "xlink:href" or "xlink:type".
XML-1: [ERROR] SAX Parser Exception: attribute "loext:font-weight" not allowed here; expected attribute "xlink:actuate", "xlink:href" or "xlink:type".
XML-1: [ERROR] SAX Parser Exception: element "svg:font-face-uri" missing required attributes "xlink:href" and "xlink:type".
XML-1: [ERROR] SAX Parser Exception: element "office:binary-data" not allowed here; expected the element end-tag or element "svg:font-face-format".
XML-1: [ERROR] SAX Parser Exception: attribute "loext:font-style" not allowed here; expected attribute "xlink:actuate", "xlink:href" or "xlink:type".
XML-1: [ERROR] SAX Parser Exception: attribute "loext:font-weight" not allowed here; expected attribute "xlink:actuate", "xlink:href" or "xlink:type".
XML-1: [ERROR] SAX Parser Exception: element "svg:font-face-uri" missing required attributes "xlink:href" and "xlink:type".
XML-1: [ERROR] SAX Parser Exception: element "office:binary-data" not allowed here; expected the element end-tag or element "svg:font-face-format".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "table:content-validation-name", "table:formula", "table:number-columns-repeated", "table:num
ber-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "table:content-validation-name", "table:formula", "table:number-columns-repeated", "table:num
ber-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "table:content-validation-name", "table:formula", "table:number-columns-repeated", "table:num
ber-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "office:string-value", "table:content-validation-name", "table:formula", "table:number-column
s-repeated", "table:number-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "office:string-value", "table:content-validation-name", "table:formula", "table:number-column
s-repeated", "table:number-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "office:string-value", "table:content-validation-name", "table:formula", "table:number-column
s-repeated", "table:number-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "office:string-value", "table:content-validation-name", "table:formula", "table:number-column
s-repeated", "table:number-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "table:content-validation-name", "table:formula", "table:number-columns-repeated", "table:num
ber-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "table:content-validation-name", "table:formula", "table:number-columns-repeated", "table:num
ber-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "table:content-validation-name", "table:formula", "table:number-columns-repeated", "table:num
ber-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "office:string-value", "table:content-validation-name", "table:formula", "table:number-column
s-repeated", "table:number-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "office:string-value", "table:content-validation-name", "table:formula", "table:number-column
s-repeated", "table:number-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "office:string-value", "table:content-validation-name", "table:formula", "table:number-column
s-repeated", "table:number-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
XML-1: [ERROR] SAX Parser Exception: attribute "calcext:value-type" not allowed here; expected attribute "office:string-value", "table:content-validation-name", "table:formula", "table:number-column
s-repeated", "table:number-columns-spanned", "table:number-matrix-columns-spanned", "table:number-matrix-rows-spanned", "table:number-rows-spanned", "table:protect", "table:protected", "table:style-name", "xhtml:about", "xhtml:content", "xhtml:datatype", "xhtml:property" or "xml:id".
No messages.
VALID, no errors and no warnings found.

```
